src/open_inwoner/components/templates/components/Contact/ContactForm.html

```diff
@@ -26,15 +26,23 @@
                 {% input form_object.question %}
 
                 {% if form_object.captcha %}
-                    <span class="label__label">
-                        {{ form_object.captcha.label }}<span class="label__label--required"> * </span>
-                    </span>
-                    <span>{{ form_object.captcha_text }}</span>
-                    <span>{{ form_object.captcha.question }}</span>
-                    {% field_as_widget form_object.captcha "input" form_id %}
+                    <div class="form__control ">
+                        <div class="captcha">
+                            <label class="label captcha__label">
+                                {{ form_object.captcha.label }}<span class="label__label--required"> * </span>
+                            </label>
+                            <div class="captcha__check">
+                                <div>
+                                    <span class="captcha__text">{{ form_object.captcha_text }}</span>
+                                    <span class="captcha__question">{{ form_object.captcha.question }}</span>
+                                </div>
+                                <span class="captcha__input">{% field_as_widget form_object.captcha "input" form_id %}</span>
+                            </div>
+                        </div>
+                    </div>
                 {% endif %}
 
-                {% form_actions primary_text=_("Verzenden") primary_icon="arrow_forward" %}
+                {% form_actions primary_text=_("Verzenden") primary_icon="arrow_forward" fullwidth=True %}
```

Comment:
Most of our 'narrow' forms now have a fullwidth submit button, so I've added that here too.